extensions/resource-deployment/src/services/azdataService.ts

```diff
@@ -68,4 +74,16 @@ export class AzdataService implements IAzdataService {
 	private async getJsonObjectFromFile(path: string): Promise<any> {
 		return JSON.parse(await this.platformService.readTextFile(path));
 	}
+
+	public async getEndpoints(clusterName: string, userName: string, password: string): Promise<BdcEndpoint[]> {
+		const env: NodeJS.ProcessEnv = {};
+		env['AZDATA_USERNAME'] = userName;
+		env['AZDATA_PASSWORD'] = password;
+		env['ACCEPT_EULA'] = 'yes';
+		let cmd = 'azdata login -n ' + clusterName;
+		await this.platformService.runCommand(cmd, { additionalEnvironmentVariables: env });
```

Comment:
If these fail are we going to get a friendly error message? Might be useful to wrap the error if one occurs with more information (like "Fetching Endpoints failed : <error message>")